sbt-cross/build.sbt

```diff
@@ -50,10 +50,12 @@ PgpKeys.pgpPassphrase in Global := Some(Array.emptyCharArray)
 
 sbtPlugin := true
 
-scalacOptions ++= Seq("-deprecation", "-language")
```

Comment:
It has been almost a year since I did this, so I don't remember exactly, but probably my rationale was that this option is supposed to be used as `-language:someLanguageFeature`. I'm not sure if specifying just `-language` caused any issues or I just decided that it does nothing and therefore removed it.